dockstore-webservice/src/main/java/io/dockstore/webservice/languages/LanguageHandlerInterface.java

```diff
@@ -141,7 +141,7 @@
      * @param dao                  used to retrieve information on tools
      * @return either a DAG or some form of a list of tools for a workflow
      */
-    String getContent(String mainDescriptorPath, String mainDescriptor, Set<SourceFile> secondarySourceFiles, Type type, ToolDAO dao);
+    Optional<String> getContent(String mainDescriptorPath, String mainDescriptor, Set<SourceFile> secondarySourceFiles, Type type, ToolDAO dao);
```

Comment:
I can't believe I'm going to say this, since I think Optional is great, and I'm always pushing people to use it, but this change will force us to update the Galaxy plugin as well, since it implements this interface. Not sure we want to go down that route. Maybe... Best to get Denis' opinion.

Reply:
Think you're thinking of `CompleteLanguageInterface. generateToolsTable` and `CompleteLanguageInterface.loadCytoscapeElements` because I suck at naming. I think this one is ok to rename/change to optional this is only used internally.

Reply:
(comment above was not taking into account Gary's comment about whether this is a good idea, just saying that we won't necessarily break code in a different repo)